cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -67,6 +67,9 @@ message CassandraFrameworkConfiguration {
      * target number of seed nodes.
      */
     optional int32 targetNumberOfSeeds = 9;
+
+    /** Default rack/dc info */
+    optional RackDc defaultRackDc = 10;
```

Comment:
Please move this property into the `CassandraConfigRole` message so that it is part of `defaultConfigRole`.